lib/bank_accounts/account.rb

```diff
@@ -0,0 +1,45 @@
+require 'pry'
+
+module BankAccounts
+  class Account
+
+    attr_reader :id, :balance
+
+    # Rounds float value down to hundredths position
+    # For example 10.2599 rounds to 10.25
+    def round_to_hundredths(n)
+      n = (((n * 100).floor) / 100.0)
+      return n
+    end
+
+    def initialize(id, initial_balance)
+      @id = id
+      @balance = round_to_hundredths(initial_balance)
+
+      # Raises an error if the initial balance is negative
+      if initial_balance < 0
+        raise ArgumentError.new "Sorry, you can't start an account with a negative balance."
+      end
+    end
+
+    def withdraw(amount)
+
+      # Raises warning if withdraw amount is greater than present balance
+      if amount > @balance
+        raise ArgumentError.new "INSUFFICIENT FUNDS\nYour current balance is $#{round_to_hundredths(@balance)}."
+
+      # Return the updated balance once money is withdrawn from the account
+      else
+        @balance -= amount
+      end
+      return @balance = round_to_hundredths(@balance)
```

Comment:
Similarly to above, the implicit return rule exists here. So at the very least you can drop the word `return`

Another question to consider. At what points in the user flow do we need to round to hundredths? If the balance coming into this method already is rounded to hundredths and a manual withdrawal or deposit can't have more significant digits than pennies, do we need to round to hundredths at the end of a manual deposit or withdrawal?